src/components/Table/EmptyTable.tsx

```diff
@@ -33,17 +33,27 @@ export default function EmptyTable() {
       : false;
   let contents = <></>;
 
-  if (tableSettings.isCollection !== false && hasData) {
+  if (
+    (tableSettings.isCollection !== false ||
+      tableSettings.subTableKey?.length) &&
```

Comment:
@rishipurwar1 how is `tableSettings.subTableKey?.length` used here, could we just remove `tableSettings.isCollection !== false` part of the condition?

Reply:
> could we just remove `tableSettings.isCollection !== false` part of the condition?

Yes, we can do that. That way, we only need to check for the `hasData` flag, if it's true then show the import existing data template.

Reply:
Updated the if condition.

Reply:
@iamanishroy what do you think? is there any other conditions we should watch out for?

Reply:
LGTM 👍